src/content/en/2020/media.md

```diff
@@ -4,13 +4,14 @@ part_number: I
 chapter_number: 5
 title: Media
 description: Media chapter of the 2020 Web Almanac covering image file sizes and formats, responsive images, client hints, lazy loading, accessibility and video.
-authors: [tpiros, bseymour]
-reviewers: [nhoizey, zcorpan, colinbendell, dougsillars]
+authors: [tpiros, bseymour, eeeps]
+reviewers: [nhoizey, colinbendell, dougsillars, Navaneeth-akam]
 analysts: [smatei]
-editors: []
+editors: [bazzadp]
 translators: []
-#tpiros_bio: TODO
-#bseymour_bio: TODO
+tpiros_bio: Tamas Piros is a Developer Experience Engineer at <a href="https://cloudinary.com/">Cloudinary</a>, Google Developer Expert and Technical Instructor.
+bseymour_bio: Ben Seymour is a Dynamic Media & Content Specialist at <a href="https://cloudinary.com/">Cloudinary</a>, author of "Practical Responsive Images".
```

Comment:
Shall we link your book @bseymour ?

```suggestion
bseymour_bio: Ben Seymour is a Dynamic Media & Content Specialist at <a href="https://cloudinary.com/">Cloudinary</a>, author of <a href="http://responsiveimag.es/">Practical Responsive Images</a>.
```

And shame on you for not being on HTTPS yet these days! 😁